### PR TITLE
i18next support

### PR DESCRIPTION
**Pull request by cacpgomes**
Add internationalization with defaults for all player buttons in Portuguese and English.

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import i18n from "i18next";
```
Comment (lijinke666) [inline src/i18n.js]: I think we don't need `i18next` and `react-i18next`
we can defined a `language.js` file 

```js
const languages = {
   en : {
       name : 'xxx'
    }
   zh : {
      name: 'xxx'
   }
}
```

in music player component 

```js
const language = languages[this.props.lng]
language.name // xxx
```
Do you think so ?
first, thank you very much for your pr
now there's a little bit too much dependency

**Comment by lijinke666** [comment]
Thank you for your pr ! that's great!
but i think we don't need 'i18next',  you can see comment
I also saw you do some compatibility work, thanks again

File: package.json [hunk 2]
```diff
@@ -6,7 +6,6 @@
   "typing": "index.d.ts",
   "files": [
     "lib/",
-    "assets/",
```
[the remark this answers is not in the document]
Comment (cacpgomes) [reply package.json]: Sorry, my mistake. I forgot the css.

File: src/index.js [hunk 3]
```diff
@@ -326,48 +335,65 @@ export default class ReactJkMusicPlayer extends PureComponent {
...
         key: 'shufflePlay',
-        value: shufflePlay,
+        value: playModeText.shufflePlay,
       },
     }
     this._PLAY_MODE_ = Object.values(this.PLAY_MODE)
     this._PLAY_MODE_LENGTH_ = this._PLAY_MODE_.length
   }
   render() {
+    i18next.changeLanguage(this.props.lng)
```
[the remark this answers is not in the document]
Comment (cacpgomes) [reply src/index.js]: Yes, componentWillReceiveprops is better. Thanks.

File: src/i18n.js (hunk 1 above)
Comment (cacpgomes) [reply src/i18n.js]: Thank you for your great component. I'm pleased to contribute to your greate work.
I'm a newby in React and I'm digging in about internationalization of a project of my own. I'm trying to find the best practices to do that and I'll apply to my contribution to your project.

**Comment by lijinke666** [comment]
I think use simple json config enough, i can adjust the feature, thanks again

**Pull request merged**